reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -602,50 +602,44 @@
   } ],
   "dependency_trees" : [ {
     "key" : 0,
-    "title" : "Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0",
+    "linkage" : null,
```

Comment:
Commit message: *Why* is it not needed anymore?

Reply:
`title` was something that was introduced as it was a needed property by Tree component of Ant Design and initially we could not figure out how to get this component to work without it. The WebApp has since been rewritten and now `title` property is generated so it's not longer needed. Removing it will make EvaluatedModel a little bit more generic and less specifically optimized for the WebApp

Reply:
Thanks @tsteenbe , I've updated the commit message.